### PR TITLE
clear logs

### PR DESCRIPTION
**Pull request by ZainVirani**
@Jkat @um4r12 please test this

### CODE REVIEW

**Comment by christinerogers** [comment]
This PR aims to remove the 3million+ log messages being committed on cap production today. 
from the apache logs: 

[Thu Aug 23 17:25:50 2018] [notice] [client 142.157.170.128] PHP Notice:  Only variables should be assigned by reference in /var/www/cap/php/libraries/NDB_BVL_InstrumentStatus_ControlPanel.class.inc on line 79, referer: https://cap.loris.ca/instrument_list/?candID=211594&sessionID=167

[Thu Aug 23 17:25:51 2018] [notice] [client 142.157.170.128] PHP Notice:  Undefined index: Name in /var/www/cap/php/libraries/NDB_BVL_Instrument_JSON.class.inc on line 275, referer: https://cap.loris.ca/instrument_list/?candID=211594&sessionID=167

File: php/libraries/NDB_BVL_InstrumentStatus.class.inc [hunk 1]
```diff
@@ -115,7 +115,8 @@ class NDB_BVL_InstrumentStatus
      */
     function getDataEntryStatus()
     {
-        return $this->_status['Data_entry'];
+        if (isset($this->_status['Data_entry'])) return $this->_status['Data_entry'];
```
Comment (christinerogers) [inline php/libraries/NDB_BVL_InstrumentStatus.class.inc:118]: Isn't it easier to ensure there is a Data_entry var? (presumably it's missing for Json instruments which should have it)

**Review by christinerogers — COMMENTED** [review]
see question about missing(?) Data_entry flag/var

**Comment by Jkat** [comment]
didn't test but looks fine.
@ZainVirani line 46 in php/libraries/NDB_BVL_InstrumentStatus_ControlPanel.class.inc already checks for `$this->tpl_data['subtest']` right

**Comment by Jkat** [comment]
Merging now but will have to fix the redundant check + `$subtest` is not even used.

**Pull request merged**